test/bulk2.test.ts

```diff
@@ -197,10 +197,11 @@ if (isNodeJS()) {
   it('should bulk insert from file and return inserted results', async () => {
     // insert 100 account records from csv file
     const csvStream = fs.createReadStream(
-      path.join(__dirname, 'data/Account_bulk2_test.csv'),
+      path.join(__dirname, 'data', 'Account_bulk2_test.csv'),
     );
 
     const res = await conn.bulk2.loadAndWaitForResults({
+      lineEnding: require('os').platform() === 'win32' ? 'CRLF' : 'LF',
```

Comment:
default value is `LF`, need to be `CRLF` when working on windows or you get a `ClientInputError`.

plugin-data does the same here:
https://github.com/salesforcecli/plugin-data/blob/85b73121d52fa02fb840b8bd2d6838bdf35ac632/src/bulkOperationCommand.ts#L112